.k8s_ci.Dockerfile

```diff
@@ -81,8 +96,31 @@ RUN rm -rf \
 # Clean up *all* node_modules, including plugins.  Everything is already built+staged.
 RUN find . -name "node_modules" -type d -prune -exec rm -rf '{}' +
 
-# Stage-2
-FROM $STAGE2_BASE
+#======================================================
+# Stage 2.2 - Build galaxy client
+#======================================================
+FROM stage1 AS client_build
+ARG SERVER_DIR
+
+RUN ansible-playbook -i localhost, playbook.yml -v --tags "galaxy_build_client"
+
+WORKDIR $SERVER_DIR
+RUN rm -rf \
+        .ci \
+        .git \
+        .venv/bin/node \
+        .venv/include/node \
+        .venv/src/node* \
+        doc \
+        test \
+        test-data
+# Clean up *all* node_modules, including plugins.  Everything is already built+staged.
+RUN find . -name "node_modules" -type d -prune -exec rm -rf '{}' +
```

Comment:
@mvdbeek  You mentioned that the expression tools need node. Is it ok if we leave just the node binary or do we need some node_modules as well? The ideal scenario would be if we could separate the node_modules needed by the expression tools from the client node_modules, so that the final image size remains small.

Reply:
No, they don't currently use any node modules.  Simply having node available as an execution context should be enough.  Down the road if we did have more complicated expression tools we could add a build process to avoid shipping all of node_modules, if necessary.